runtime/gc_vlhgc/VLHGCAccessBarrier.cpp

```diff
@@ -271,6 +271,19 @@ MM_VLHGCAccessBarrier::copyArrayCritical(J9VMThread *vmThread, GC_ArrayObjectMod
 	}
 }
 
+IDATA
+MM_VLHGCAccessBarrier::indexableDataDisplacement(J9VMThread *vmThread, J9IndexableObject *src, J9IndexableObject *dst)
+{
+	IDATA displacement = 0;
+
+	Assert_MM_true(vmThread->isVirtualLargeObjectHeapEnabled);
+	/* Adjacency check against dst object since src object may be overwritten during sliding compaction. */
+	if (_extensions->indexableObjectModel.isDataAdjacentToHeader(dst)) {
+		displacement = (IDATA) (((UDATA)dst) - ((UDATA)src));
```

Comment:
you can also remove the space after first IDATA cast (here and in the base class)
if fact you can call base class variant of this method

Reply:
and base class one can be inlined (will not prevent the initial virutal dispatch, but should inline it here)